website/docs/reference/resource-configs/plus-prefix.md

```diff
@@ -62,3 +62,9 @@ models:
 </File>
 
 Since it doesn't hurt to use the `+` prefix, we recommend you use it whenever adding configs to your `dbt_project.yml` file.
```

Comment:
perhaps worth being more specific bc not really recommended to put prefix all out grant configs w/ plus?
```suggestion
Since it doesn't hurt to use the `+` prefix for configuration options, we recommend you use it whenever adding configs to your `dbt_project.yml` file.
```